translation/en/RedirectedResources/assets/abdata/h/list/00_00/personality_voice_c01_03_00/translation.txt

```diff
@@ -1013,7 +1013,7 @@
 おまんこ、まだヒクヒクしています=My pussy, is still twitching.
 本当に、癖になってしまいそう……=I really might, get addicted to this...
 もう、激しくしすぎですよぉ……=Jeez, you went way too hard there, dummy~...
-イかされる＝愛されていると、最近は感じるんです=Lately, in my head, it's all been "He makes me cum
+イかされる＝愛されていると、最近は感じるんです=Lately, in my head, it's all been "He makes me cum ＝ He loves me".
```

Comment:
This works because it's using wide `＝` instead of `=`, but I wonder if using `\=` would be more obvious.

```suggestion
イかされる＝愛されていると、最近は感じるんです=Lately, in my head, it's all been "He makes me cum \= He loves me".
```

Reply:
I believe the wide character trick was used in a few places. It's used only when the original text also uses the wide version so I think it's fine, probably. I don't think anyone will touch these translations after this point anyways.